Plugins/FormatSwift/Plugin.swift

```diff
@@ -20,16 +20,21 @@ struct AirbnbSwiftFormatPlugin: CommandPlugin {
     // specifying the targets selected in the plugin dialog.
     let inputTargets = argumentExtractor.extractOption(named: "target")
 
+    // Only lint the paths passed to `--paths`
+    let strictPaths = argumentExtractor.extractOption(named: "paths")
```

Comment:
Having a single `inputPaths` property and then doing `inputPaths += targetPaths` below may be a bit more idiomatic

```suggestion
    var inputPaths = argumentExtractor.extractOption(named: "paths")
```